tests/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+#
+# Development configuration of dependent services for Docker Compose.
+#
+postgresql:
+    image: postgres:9.4
+    environment:
+        POSTGRES_USER: resolwe
+        POSTGRES_DB: resolwe
+    ports:
+        - "55432:5432"
```

Comment:
Yes, I see your point. I suggest that you, as an advanced developer, export the `RESOLWE_POSTGRESQL_PORT` environment variable to the desired port and only run one database instance. For the rest, it will be more friendly to have the configurations not clashing.

Following our discussion, I'll rather change the port in Resolwe Bioinformatics to `55433` and leave this one at `55432`.